components/Modal/Modal.js

```diff
@@ -30,6 +33,7 @@ function Modal({
   onRequestClose,
   screenReaderLabel,
   shouldCloseOnOverlayClick,
+  isContainerScrollable,
```

Comment:
Instead of this, let's do a `childrenClassName` prop so that we can override any styles passed in via `ModalStyles.scrollableContainer`

Reply:
Just so I am clear, are you suggesting something like this?

```jsx
<div className={classNames(ModalStyles.scrollableContainer, childrenClassName) data-testid={SCROLLABLE_CONTAINER}>
   {children}
</div>
```

Reply:
I actually pulled your branch down and - after a few tweaks - found it no longer necessary to edit the Modal component at all.

Where I ended up at:
```jsx
<Modal
  isOpen={isModalOpen}
  screenReaderLabel="Login Modal"
  onRequestClose={closeModal}
  className={CardStyles.CardModal}
>
  <h2>Login to Proceed</h2>

  <LoginForm
    login={handleLogin}
    onSuccess={handleLoginSuccess}
    redirectFunc={() => {
      closeModal();
      router.push(router.asPath);
    }}
  />

  <p>
    Forgot your password? Reset it&nbsp;
    <Link href="/password_reset">
      <a>here</a>
    </Link>
  </p>
</Modal>
```

Quick aside, I'm very sorry about the confusing nature of the LoginForm props. I'll have to address that.

Reply:
Okay. I am guessing you modified the modal component itself as well? For now, I'll apply the change you made here. Let me know if any additional change's needed.

Reply:
Yes, I reverted all changes to the modal compoenent.